darwin/future/core/client.py

```diff
@@ -186,7 +186,7 @@ def _contain_qs_and_endpoint(self, endpoint: str, query_string: Optional[QuerySt
             return endpoint
 
         assert "?" not in endpoint
-        return endpoint + "?" + str(query_string)
+        return endpoint + str(query_string)
```

Comment:
Found this as I was going, it was adding ? in two locations.  Thank you social testing.